### DISCUSSION TITLE
install problems

### DISCUSSION THREAD

**Discussion by Darkhell101** [alandefreitas/matplotplusplus#214] [Q&A]
When I install source code and use the command "cmake ." "make", It works.
And I don't know how to use the binary package if I install it in /usr/local/include/.

**Comment by alandefreitas** [comment]
> When I install source code and use the command "cmake ." "make", It works.

If you want to use the source code directly, there's nothing to install. You can just copy and paste the source code. If you're using CMake with add_subdirectory, that should be good enough.

> And I don't know how to use the binary package if I install it in /usr/local/include/.

If you are installing the binaries (not the source code), then you're supposed to `find_package` with CMake

**Reply by Darkhell101** [reply]
thanks, I will try

**Reply by Darkhell101** [reply]
I run the examples successfully
